device-types/DataDirect Networks/SFA7990X-HDR100.yaml

```diff
@@ -0,0 +1,65 @@
+---
+manufacturer: DataDirect Networks
+model: SFA7990X-HDR100
+slug: sfa7990x-hdr100
```

Comment:
All slugs should have the manufacturer appended. In this case, please append `datadirect-networks-`